openedx/core/djangoapps/profile_images/tasks.py

```diff
@@ -0,0 +1,27 @@
+"""
+This file contains celery tasks for profile_images
+"""
+from __future__ import absolute_import
+
+from celery.task import task
+from celery.utils.log import get_task_logger
+
+from ..user_api.accounts.image_helpers import get_profile_image_names
+from .images import remove_profile_images
+
+
+LOGGER = get_task_logger(__name__)
+
+
+@task
+def delete_profile_images(usernames):
+    """
+    Delete profile images from storage belonging to usernames.
+
+    Arguments:
+        usernames: list of usernames
+    """
+    for username in usernames:
+        profile_image_names = get_profile_image_names(username)
+        LOGGER.info('Deleting profile images for %s...', username)
+        remove_profile_images(profile_image_names)
```

Comment:
@john2x, in case deleting the profile image of a particular user fails, that would cause the deletion of the remaining usernames in the list to be skipped. Does it make sense to handle the exception, log it and proceed?

Reply:
@lgp171188 ah yes, good catch. I'll update that.

Reply:
@lgp171188 Updated